x/ccv/consumer/types/params.go

```diff
@@ -52,41 +51,35 @@ func DefaultParams() Params {
 
 // Validate all ccv-consumer module parameters
 func (p Params) Validate() error {
+	if err := ccvtypes.ValidateBool(p.Enabled); err != nil {
+		return err
+	}
+	if err := ccvtypes.ValidatePositiveInt64(p.BlocksPerDistributionTransmission); err != nil {
+		return err
+	}
+	if err := ccvtypes.ValidateString(p.DistributionTransmissionChannel); err != nil {
```

Comment:
Maybe use `ChannelIdentifierValidator` https://github.com/cosmos/ibc-go/blob/f693f6c11053192f369b5908c9186bd494c3a33d/modules/core/24-host/validate.go#L79

Reply:
I agree that this is a better validator than just *is this a string?*

However, there are places all over tests that we set the value of `DistributionTransmissionChannel` to an empty string. In fact the[ default parameters ](https://github.com/cosmos/interchain-security/blob/params-validation/x/ccv/consumer/types/params.go#L42) for a consumer has that field as empty. I'm not entirely aware of the intricacies of this param, so is there a better default value we can use?

Btw same issue applies to your comment below and `ProviderFeePoolAddrStr`

Reply:
It might be the case that we only want to properly validate these parameters where they're properly set [here](https://github.com/cosmos/interchain-security/blob/params-validation/x/ccv/consumer/ibc_module.go#L102)

Reply:
See new `validateDistributionTransmissionChannel`